PublicGitArchive/pga/pga/pga.go

```diff
@@ -154,10 +164,17 @@ func IndexFromCSV(r io.Reader) (Index, error) {
 		return nil, fmt.Errorf("could not skip headers row: %v", err)
 	}
 
-	withStars := true
+	var (
+		withStars     = true
+		withSivasSize = true
+	)
+
 	switch {
-	case len(header) == len(csvHeaders)-1:
+	case len(header) == len(csvHeaders)-2:
```

Comment:
I would check for these columns using the headers. If there's a `STARS` header then it has stars. It's position in the header also gives the column used to read it. Same for size.